drltrace/CMakeLists.txt

```diff
@@ -133,29 +137,45 @@ if (BUILD_TOOL_TESTS)
   get_target_path_for_execution(drltrace_path drltrace)
   get_target_path_for_execution(app_path drsyscall_app)
   prefix_cmd_if_necessary(drltrace_path OFF ${drltrace_path})
-
   add_test(drltrace ${drltrace_path} -- ${app_path})
   add_test(drltrace_libcalls ${drltrace_path} -logdir - -print_ret_addr -- ${app_path})
   add_test(drltrace_symargs ${drltrace_path} -logdir - -num_max_args 4 -- ${app_path})
 
+  set(path_to_config ${INSTALL_BIN}/drltrace.config)
+
+  # copy drltrace.config required to test libcall arguments printing
```

Comment:
My point here is that this comment implies that you only copy the config file for your test, when it's needed for regular execution and not just the test.